src/main/java/com/commercetools/sync/inventories/InventorySync.java

```diff
@@ -77,9 +79,7 @@ public InventorySync(@Nonnull final InventorySyncOptions syncOptions) {
 
     /**
      * Iterates through the whole {@code inventories} list and accumulates its valid drafts to batches. Every batch
-     * is then processed by {@link InventorySync#processBatch(List)}. For invalid drafts from {@code inventories}
-     * "processed" and "failed" counters from statistics are incremented and error callback is executed. A valid draft
-     * is a {@link InventoryEntryDraft} object that is not {@code null} and its SKU is not empty.
+     * is then processed by {@link InventorySync#processBatch(List)}.
```

Comment:
why remove the doc about the stats being updated? It is also a side effect of calling this method, no?